apps/frontend/src/modules/chat/libs/helpers/check-is-task.helper.ts

```diff
@@ -0,0 +1,9 @@
+import { type ChatMessageDto, type TaskMessage } from "../types/types.js";
+
+const checkIsTask = (
```

Comment:
```suggestion
import { type ChatMessageDto, type TaskMessage } from "../types/types.js";

const checkIsTaskMessage = (
```
Don't forget about the file name